README.md

```diff
@@ -14,5 +14,5 @@ Front end client
 #### Linux
 The following packages are required to build locally.
 ```shell
-sudo apt install libasound2-dev libudev-dev
+sudo apt install pkg-config libasound2-dev libudev-dev libwayland-dev libxkcommon-dev
```

Comment:
```suggestion
sudo apt install g++ pkg-config libx11-dev libasound2-dev libudev-dev libwayland-dev libxkbcommon-dev
```
You may also need Vulkan drivers for your graphics card.

Reply:
You may also need Vulkan drivers for your graphics card.
Source: https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md